server/src/routes/api/payments.js

```diff
@@ -0,0 +1,8 @@
+const router = require('express').Router()
```

Comment:
you can delete this from  your branch :)

Reply:
NP